media-libs/avidemux-core/avidemux-core-2.6.20.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit cmake-utils
+
+SLOT="2.6"
+
+DESCRIPTION="Core libraries for a video editor designed for simple cutting, filtering and encoding tasks"
+HOMEPAGE="http://fixounet.free.fr/avidemux"
+
+# Multiple licenses because of all the bundled stuff.
+LICENSE="GPL-1 GPL-2 MIT PSF-2 public-domain"
+IUSE="debug nls nvenc sdl system-ffmpeg vaapi vdpau video_cards_fglrx xv"
+KEYWORDS="~amd64 ~x86"
+
+if [[ ${PV} == *9999* ]] ; then
+	KEYWORDS=""
```

Comment:
remove this completely (you only ever want one `KEYWORDS=` line in an ebuild for `ekeyword` not to go nuts)